themes/mojada.omp.json

```diff
@@ -0,0 +1,152 @@
+{
+    "blocks": [{
+            "type": "prompt",
+            "alignment": "left",
+            "newline": true,
+            "segments": [{
+                    "type": "os",
+                    "style": "diamond",
+                    "leading_diamond": "",
+                    "trailing_diamond": "",
+                    "foreground": "#0077c2",
+                    "background": "#fbfbfb",
+                    "properties": {
+                        "prefix": "",
+                        "postfix": " ",
+                        "macos": "mac",
+                        "windows": ""
```

Comment:
Why would we add a blank space for Windows?

Reply:
That was my fault because I copy the character instead of directly reference to its unicode. I will fix them all in the next commit